src/components/tile/tile.config.ts

```diff
@@ -8,3 +8,11 @@ export const TILE_BORDER_VARIANTS = [
   "positive",
   "caution",
 ];
+export const TILE_HIGHLIGHT_VARIANTS = [
```

Comment:
I went back and forth about whether we should match the above border variant array elements, ticket steered to use Message variants but it feels a bit weird having the mismatch between the border and highlight variants etc (they do use slightly different tokens)